Source/OnlineSubsystemEpic/Private/OnlineUserInterfaceEpic.cpp

```diff
@@ -672,9 +681,11 @@ bool FOnlineUserEpic::GetAllUserInfo(int32 LocalUserNum, TArray< TSharedRef<clas
 
 						TSharedRef<FUserOnlineAccount> localUser = MakeShared<FUserOnlineAccountEpic>(epicNetId);
 						localUser->SetUserAttribute(USER_ATTR_COUNTRY, country);
+						localUser->SetUserAttribute(USER_ATTR_REALNAME, displayName);
```

Comment:
See comment on #42

Reply:
Merge in user info query first and this should be fixed.